communications.md

```diff
@@ -0,0 +1,68 @@
+
+For synchronous communications:
+
+1. Aggressively timebox them. If a meeting was set to be an hour-long, ask if it can be shortened to 30 minutes; if it was set to be 30 minutes long, ask if it can be shortened to 15 minutes...
+2. Write down the agenda, and prepare every participant.
+3. Instead of abstractly talk about a problem in a hollow boardroom, demonstrate the problem, perhaps on-site or screen-shared (which can be your local branch of a project that you're working on). Do not stretch beyond the agreed timebox, make a decision and move on.
+
+## Minimize Processes and Tools
+
+Processes and tools are supposed to serve us and to save us from events that undermine our productivity or even help to boost it. But on many occasions, these processes and tools become a burden on the makers. Instead, they merely become things for backline delegators to flirt with, to feel good and feel important about themselves. Or they become a means for those that are not good or have no interest at their job to deviate from real tasks, to show that they have poured in endless hours on the planning and blueprints and thus worthy of the pretended job security.
+
+Thus processes such as SCRUM, and tools such as Slack, which originally set out to solve real problems, have become things that people criticize or even hate. Because maintaining discipline for every participant of a tool or a process is an astonishingly hard task.
+
+Everyone has a different way of working and thinking. This fact should be embraced, instead of introducing rules and processes to resist or neglect it.
+
+Therefore, try to minimize processes and tools. Before you demand for a tool that can nicely map out your product's user workflow, use a stack of post-it and some colored markers, map it out on a virtual or physical whiteboard; instead of putting the effort to shop for a progress tracking tool that can make you happy (or miserable) about yourself at the end of a working day, how about start with writing the todos on a piece of paper.
+
+It's no wonders that forward-thinking managers at larger organizations even dare to bypass settled processes, so that teams can install a newer version of an operating system that's necessary for development, instead of waiting for the process to eventually get the IT department to have it done, perhaps after 3 months.
+
+## ~~"Inbox Zero~~ Everything Zero"
+
+A busy email inbox is not the only contributing factor to interruptions. Nowadays we have way too many channels of receiving information, by choice or not.
+
+So instead of achieving "Inbox Zero", try to be as close as possible to "Everything Zero". This means diligently clear or defer any information that is not work-related. Some platforms offer one simple toggle to enter the "Do Not Disturb" mode. Use it. Clear up your desktop, notification stacks, browser windows/tabs, etc. This helps to minimize distractions, so you can find what you need to focus on, faster.
```

Comment:
Remove the "So".
Suggestion:
`Instead of striving to achieve "Inbox Zero",....`